packages/nimble-components/src/table/models/virtualizer.ts

```diff
@@ -25,6 +25,10 @@ export class Virtualizer<TData extends TableRecord = TableRecord> {
     @observable
     public scrollHeight = 0;
 
+    public get isScrolling(): boolean {
```

Comment:
I had to make this a property with a getter (rather than just a data field) to be able to mock it with `jasmine.spyOnProperty`. I then found that `@observable` doesn't seem to work on a property with only a getter, so I just added an observable, private field.